docs/supabaseConnect.md

```
@@ -0,0 +1,67 @@
+    supabase link --project-ref <project-ref>
+```
+
+Replace <project-ref> with your Supabase project reference ID, which can be found in your Supabase dashboard.
+
+5.  **Create a Migration**
+
+```bash
+    npx supabase migration new <migration_name>
+```
+
+### Example:
```

Markdownlint (0.35.0)

Expected: h2; Actual: h3
Heading levels should only increment by one level at a time

(MD001, heading-increment)